Makefile

```diff
@@ -26,9 +28,10 @@ prep_version_incrementor:
 	@pipenv --python $(PYTHON_VERSION)
 	@pipenv run pip install -i https://artefacts.tax.service.gov.uk/artifactory/api/pypi/pips/simple version-incrementor==0.2.0
 
-clean: ## Remove the fluentbit docker image
+clean: ## Remove the docker image
 	@echo '********** Cleaning up ************'
 	@docker rmi -f $$(docker images artefacts.tax.service.gov.uk/$(COMPONENT_RENDERER_SERVICE_NAME):$$(cat .version) -q)
+	@docker rmi -f $$(docker images artefacts.tax.service.gov.uk/$(COMPONENT_RENDERER_SERVICE_NAME):latest -q)
```

Comment:
Elsewhere I've just used something like `sh "docker rmi -f \$(docker images -q)"` to get rid of everything